src/escrypt/scrypt_platform.c

```diff
@@ -73,10 +72,10 @@ static int
 free_region(escrypt_region_t * region)
 {
 	if (region->base) {
-#if defined(MAP_ANON) && !defined (MEMDBG_ON)
+#if defined(MAP_ANON)
 		if (munmap(region->base, region->size))
 			return -1;
-#elif defined(HAVE_POSIX_MEMALIGN) && !defined (MEMDBG_ON)
+#elif defined(HAVE_POSIX_MEMALIGN)
 		libc_free(region->base);
```

Comment:
I'm pretty sure libc_free is memdbg stuff. It should be MEM_FREE

Reply:
Hmm, I thought I caught all of them.  I must have used grep here, and missed this one.  Will fix.